tests/unit/src/test/scala/tests/NewFilesLspSuite.scala

```diff
@@ -100,15 +101,36 @@ class NewFilesLspSuite extends BaseLspSuite("new-files") {
                        |""".stripMargin
   )
 
+  val filePath = "a/src/main/scala/foo/Other.scala"
+  val existingFileContents =
+    """|package foo
+       |
+       |class Other {
+       | val withContent = true
+       |}
+       |""".stripMargin
```

Comment:
these two declarations look a bit out here, among `check`s. I would put it somewhere down with appropriate names (fixture style), or remove a new line between them and the check below, to express they are related.

Reply:
I moved them into the check, I think it's fine to have it duplicated there and looks more like the other test cases.